sfdx-project.json

```diff
@@ -5,7 +5,7 @@
       "default": true
     }
   ],
-  "namespace": "fHCM2",
+  "namespace": "",
```

Comment:
are we running this test in no namespace org .. or in the workflow when we create the org you want to mention the namespace as -D parameter ?

Reply:
The tests run without a namespace, which matches the origin. However, the code has been modified to work when it is deployed into an org and package that has a namespace.